src/app/portlets/dot-edit-page/content/services/dot-edit-content-html/dot-edit-content-html.service.ts

```diff
@@ -387,10 +387,38 @@ export class DotEditContentHtmlService {
         this.dotEditContentToolbarHtmlService.addContentletMarkup(doc);
     }
 
+    private setScriptProperties(script: HTMLScriptElement, node: any): HTMLScriptElement {
+        script.type = 'text/javascript';
+
+        if (node.src) {
+            script.src = node.src;
+        } else {
+            script.text = node.textContent;
+        }
+        return script;
+    }
+
+    private getScriptTags(scriptTags, contentDivWrapper: HTMLElement) {
+        const doc = this.getEditPageDocument();
+
+        Array.from(contentDivWrapper.children).forEach((node: any) => {
+            if (node.tagName === 'SCRIPT') {
+                let script = doc.createElement('script');
+                script = this.setScriptProperties(script, node);
+                scriptTags.push(script);
```

Comment:
why not create a bigger method like:

cretaeSciprtTag() {
    const doc = this.getEditPageDocument();
   let script = doc.createElement('script');
   script = <code of setScriptProperties>; //in this case we don't need the setScriptProperties method, we 
}